### PR TITLE
HAMBURGER MENU PROJECT

### PR DESCRIPTION
**Pull request by 3nityGitHub**
## In this PR:
- We created a new branch for the project
- We made adjustments to the hamburger menu
- we added hover effects to the hamburger menu.

### CODE REVIEW

**Review by LionRouge1 — COMMENTED** [review]
Hi @3nityGitHub, 

done

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has not linter set up correctly. Kindly, fix linter errors and resubmit your project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: styles.css [hunk 1]
```diff
@@ -409,6 +409,10 @@ a {
   margin: 3px;
 }
 
+.nav_menu2 a:hover {
+  color: #f55800;
+}
+
```
Comment (Ibizugbe) [inline styles.css:415]: Nice work on implementing the functionalities of the `nav` 👍👍 Kindly note that your project looks a bit different from the given design😉 Kindly see the comparison below and take note of the colors and implement the necessary change😉 as we are required to stick to the design in this milestone👍

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|yours|design menu|

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @3nityGitHub 👋👋, 

tryagain


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅✅

- [x] Linters are properly setup and they are passing👍.
- [x] The functionalities of the mobile menu have been implemented👍.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (3nityGitHub) [reply styles.css:415]: Hello @Ibizugbe Thanks for your review but i am not sure i get you correctly. Dont you think the difference is from our different screen? I have checked the colors in figma and its the same with mine.

**Review by LionRouge1 — APPROVED** [review]
Hi @3nityGitHub, 

# STATUS: APPROVED! 🥇 💯 
Great work on making the changes 👏 
You've done well implementing the requested changes

done

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

[OPTIONAL] 
- [ ] Kindly make sure your mobile menu doesn't scroll with the page

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**